packages/gestalt/src/Contents.css

```diff
@@ -7,7 +7,7 @@
   composes: borderBox from "./Layout.css";
   composes: rounded from "./Borders.css";
```

Comment:
@dangerismycat Should we also update the border radius? Was there a reason for splitting these changes out?